packages/flutter/lib/src/material/material.dart

```diff
@@ -100,6 +101,14 @@ class Material extends StatefulComponent {
   }
 
   _MaterialState createState() => new _MaterialState();
+
+  void debugFillDescription(List<String> description) {
```

Comment:
Yeah, probably. They independently evolved out of different cruder debugging tools and have been slowly converging to the same solution. Might as well finish it. :-)